src/glyphnote.ts

```diff
@@ -36,21 +40,39 @@ export class GlyphNote extends Note {
     return this.glyph.getBoundingBox();
   }
 
-  /*
-  addToModifierContext() {
+  // Add self to modifier context. `mContext` is the `ModifierContext`
+  // to be added to.
+  addToModifierContext(mContext: ModifierContext): this {
```

Comment:
Is this necessary? Isn't it derived from the inheritance chain?

Reply:
The inherited one (`tickable`) doesn't add modifiers to the context. 

We could add this loop code to `Note`, then `stavenote` and `glyphnote` could share, but we'll have to check all the other `Note` children.  We could modify the `tickable` code to check for modifiers, maybe.

Reply:
Welp, not exactly the same.  Glyphnote doesn't need to `format`, so it doesn't add itself to the context.  So stavenote looks like this, not sure how you feel about `super`.

``` javascript
    this.modifierContext.addMember(this);
    return super.addToModifierContext(mContext);
```

Reply:
welp, this doesn't work, I'd have to add a check for null to `this.modifierContext` which seems to defeat the purpose.  Or add an empty format method for glyphnote.  

So I think I'm back to:  it is necessary :)

```
ERROR in C:\Users\Daddy\Documents\GitHub\vexflow\src\stavenote.ts
./src/stavenote.ts 871:4-24
```

Reply:
kk, thanks for digging in :-)